conans/model/conf.py

```diff
@@ -40,6 +40,9 @@
     "tools.system.package_manager:sudo": "Use 'sudo' when invoking the package manager tools in Linux (False by default)",
     "tools.system.package_manager:sudo_askpass": "Use the '-A' argument if using sudo in Linux to invoke the system package manager (False by default)",
     "tools.apple.xcodebuild:verbosity": "Verbosity level for xcodebuild: 'verbose' or 'quiet",
+    "tools.apple:enable_bitcode": "(boolean) Enable/Disable Bitcode Xcode flags",
```

Comment:
Are those Xcode exclusively? I think they are ``apple-clang`` compiler flags. Maybe change description of conf?